### PR TITLE
Remove legacy file

### PR DESCRIPTION
**Pull request by mariushegele**

### CODE REVIEW

**Comment by Ovakefali13** [comment]
Calservice and Worksession tests fail, but I don't know what seems to be the problem

**Comment by mariushegele** [comment]
I configured Travis to run builds only on PRs and to run only one job concurrently. Simultaneously running jobs were failing because they used the same calendar remote.

File: services/cal/test/test_service.py [hunk 1]
```diff
@@ -49,7 +49,6 @@ def setUpClass(self):
             self.cal_service = CalService.instance(CalMockRemote.instance())
             print('Mocking Remote...')
 
-    @classmethod
     def setUp(self):
```
Comment (Ovakefali13) [inline services/cal/test/test_service.py]: Why did you delete this ?

Comment (mariushegele) [reply services/cal/test/test_service.py]: I had read that it's only necessary for `setUpClass` and`tearDownClass` 
https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp
